### PR TITLE
Add c++20 to enums

### PR DESCRIPTION
**Pull request by Jasdriel**
This are the files in the Typescript part that I found to add the enums, is ok with this changes or there are other parts to update?

For issue #3448.

### CODE REVIEW

File: Extension/c_cpp_properties.schema.json [hunk 1]
```diff
@@ -37,6 +37,8 @@
                             "c++11",
                             "c++14",
                             "c++17",
+                            "c++20",
+                            "c++Latest",
```
Comment (sean-mcmanus) [inline Extension/c_cpp_properties.schema.json]: We shouldn't use c++20 latest -- it should just be c++20.

**Review by sean-mcmanus — CHANGES_REQUESTED** [review]
see my other comment

**Comment by michelleangela** [comment]
Also add "c++20" as another `<option>` to Config UI in [settings.html](https://github.com/microsoft/vscode-cpptools/blob/53107bc57ca9c0e772a8d73746e1caf4612568db/Extension/ui/settings.html#L562)

And to [package.json](https://github.com/microsoft/vscode-cpptools/blob/master/Extension/package.json#L421)

**Comment by sean-mcmanus** [comment]
The msvc version needs to be changed in the .json files: https://github.com/microsoft/vscode-cpptools/blob/master/Extension/bin/msvc.32.intel.json and the other 2 .msvc files.

File: Extension/c_cpp_properties.schema.json (hunk 1 above)
Comment (Jasdriel) [reply Extension/c_cpp_properties.schema.json]: ok, fixed

**Comment by sean-mcmanus** [comment]
Okay, this seems good, but don't check it in until the C++ side is checked in.

**Review by sean-mcmanus — APPROVED** [review]

**Comment by michelleangela** [comment]
Also need to add c++20 to the API https://github.com/microsoft/vscode-cpptools-api. You'll have to create separate pull request on that repo.

**Pull request merged**